interface/backend/submission.py

```diff
@@ -23,3 +42,11 @@ def handle_submission(file, assignment, user):
     submission.evaluate()
     log.debug(f'Submission #{submission.id} was sent to VMCK '
               f'as #{submission.vmck_job_id}')
+
+
+class TooManySubmissionsError(Exception):
+    def __init__(self, delta_t):
+        self.delta_t = delta_t
+
+    def get_delta_t(self):
+        return self.delta_t
```

Comment:
It's overkill to define a getter for a field that you just know is there. Just use the attribute.